wikidata.lua

```diff
@@ -17,6 +17,7 @@ function wikidata (str)
             return str:sub(2)
         end
     end
+    return nil
```

Comment:
This isn't a behavior change, and I don't think it adds any clarity. Functions reaching the end of execution and a default return is a fairly well understood concept.